app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached_data = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
Wrapper can return Callable but not always. Think about what is can be stored at your dict, you can use this decorator with any function (that functions can return int, float, list, dict, tuple, set, etc.), and you return result of that functions